### PR TITLE
add new cpu article

### PR DESCRIPTION
**Pull request by stephamon**
update main CPU spec article and add new child page

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-26e4c31e3e/how-to/)

**Comment by cyrichardson** [comment]
@stephamon Here are suggestions for the new article: 
[firmware-microcode-mitigations-against-cpu-speculative-execution-vulnerabilities_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1895791/firmware-microcode-mitigations-against-cpu-speculative-execution-vulnerabilities_cyrEdits.docx)

**Comment by cyrichardson** [comment]
@stephamon For the addition to the existing article, I am going to make comments inline in the PR. I will let you know when I'm done.

File: content/general/rackspace-mitigations-against-cpu-speculative-execution-vulnerabilities.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ This page lists various Rackspace services and their current mitigation status f
...
+| Firmware / Microcode Updates | Rackspace has received a subset of server platform firmware/microcode updates. Each firmware/microcode update will be tested and released according to our standard procedures.<br/><br/>As firmware/microcode updates become available, Rackspace will send tickets to customers who use the specific platforms. Customers who elect to patch their dedicated servers must opt in to a maintenance window to apply the update. Additional actions might be required to fully remediate against these vulnerabilities.<br/><br/>For more information, see [Firmware/Microcode mitigations against CPU speculative execution vulnerabilities](/how-to/firmware-microcode-mitigations-against-cpu-speculative-execution-vulnerabilities/). |
```
Comment (cyrichardson) [inline content/general/rackspace-mitigations-against-cpu-speculative-execution-vulnerabilities.md]: **Firmware / Microcode updates** (Make bold and initial lower case for "updates")

**must** opt in (Make must bold.)

**Comment by cyrichardson** [comment]
@stephamon A couple of inline comments in the addition to the existing article. (I'm all done now.)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-82a663b98a/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6b705cb3a6/how-to/)

**Pull request merged**